### PR TITLE
HVG-306 Hide Switcher flow from Danish web onboarding

### PR DESCRIPTION
**Pull request by virrios**
Just hide the switcher flow when the market is Denmark

### CODE REVIEW

**Review by styrelius — COMMENTED** [review]
Nice! 🌸 I guess we should also remove the "Current Insurer" input from the debugger though? I think that could be within the scope for this ticket.

**Comment by virrios** [comment]
> Nice! 🌸 I guess we should also remove the "Current Insurer" input from the debugger though? I think that could be within the scope for this ticket.

**Comment by virrios** [comment]
I saw that it was optional in the debugger. It was not obvious for me that it should be removed from the ticket description so If you think it should be removed I will do it :D

**Review by palmenhq — DISMISSED** [review]
🥳

**Comment by styrelius** [comment]
> I saw that it was optional in the debugger. It was not obvious for me that it should be removed from the ticket description so If you think it should be removed I will do it :D

No, I know - the ticket didn't say so! But since we don't support the switching for Danish quotes you will get an error if you add something to that input. So I think we should just get rid of it!

File: src/client/pages/Debugger/components/Offer.tsx [hunk 1]
```diff
@@ -218,21 +218,18 @@ export const Offer: React.FC<OfferProps> = ({ sessionToken }) => {
...
-                        placeholder=""
-                        {...props.getFieldProps('ssn')}
-                      />
+                      {currentMarket === Market.Se && (
+                        <InputField
+                          label="ssn"
+                          placeholder=""
+                          {...props.getFieldProps('ssn')}
+                        />
+                      )}
```
Comment (styrelius) [inline src/client/pages/Debugger/components/Offer.tsx:232]: Wasn't the exact same change in a previous, already merged, PR though? 🤔

**Pull request closed without merging**